BookmarksTodayExtension/TodayViewController.swift

```diff
@@ -44,12 +49,10 @@ class TodayViewController: UIViewController, NCWidgetProviding, UITableViewDeleg
     }
 
     private func configureWidgetSize() {
-        if #available(iOSApplicationExtension 10.0, *) {
-            let mode = bookmarks.count > 2 ? NCWidgetDisplayMode.expanded : NCWidgetDisplayMode.compact
-            extensionContext?.widgetLargestAvailableDisplayMode = mode
-        }
+        let mode = bookmarks.count > 2 ? NCWidgetDisplayMode.expanded : NCWidgetDisplayMode.compact
+        extensionContext?.widgetLargestAvailableDisplayMode = mode
 
-        if #available(iOSApplicationExtension 10.0, *), extensionContext?.widgetActiveDisplayMode == NCWidgetDisplayMode.compact {
+        if extensionContext?.widgetActiveDisplayMode == NCWidgetDisplayMode.compact {
             updatePreferredContentHeight(height: defaultHeight)
         } else {
             updatePreferredContentHeight(height: preferredHeight)
```

Comment:
Looking at `preferredHeight` method, it uses `let headerHeight = CGFloat(54.0)` const to adjust view height. Problem is, that with '54', expanded widget is a bit too big :)

I also wonder about defaultHeight method with 110 constant - we should probably be using [widgetMaximumSize(for:)](https://developer.apple.com/documentation/foundation/nsextensioncontext/1649135-widgetmaximumsize) there. One way to see problem around it is to change preferred font size in accessibility settings.

Reply:
Good point - those hardcoded values aren't nice (and some of them come from the initial hack day project). I've updated the code to remove them.